test: update white list and correct links in docs closes #832 #833

fozy81 opened the pull request.

fozy81 committed:
test: update white list and coorect links in docs

fozy81 commented:
closes #833 #832

fozy81 committed:
test: remove awesome_bot and use textlint instead

fozy81 commented:
updated the PR to remove dependency on awesome_bot and set dead link tests to warnings only so won't break the build.

fozy81 committed:
test: remove dead link testing - not working correctly

gr2m commented on .travis.yml:
```diff
@@ -3,22 +3,17 @@ language: node_js
   email: false
 node_js:
-  - 4
+  - 6
```
Does it fail in Node 4? Not a big deal, I planned to suggest to align Node dependency with hapi anyway (which is 8 now), and I think requiring 6 is reasonable, just wondering

fozy81 committed:
test: removing dead-link-checker

fozy81 committed:
test: returning node to version 4 as textlint-rule-no-dead-link dev has been removed and node 6 not required

fozy81 commented:
The upping the node version was suggested as a [fix](https://github.com/textlint-rule/textlint-rule-no-dead-link/issues/81#issuecomment-353582093) to a problem with using textlint-rule-no-dead-link but now that I've removed this dependency (also removed it from package.json just now). And I've returned node version 4. So leave it with you to update to version 8. Sorry - went through a few options trying to get things to work this morning and forgot I'd changed the node version.

gr2m reviewed (approved):
thank you :)
